sdk/testing/src/test/java/io/opentelemetry/sdk/testing/assertj/LogAssertionsTest.java

```diff
@@ -256,24 +270,71 @@ void failure() {
                         attributes ->
                             OpenTelemetryAssertions.assertThat(attributes)
                                 .hasEntrySatisfying(
-                                    AttributeKey.stringKey("bear"),
-                                    value -> assertThat(value).hasSize(2))))
+                                    stringKey("bear"), value -> assertThat(value).hasSize(2))))
         .isInstanceOf(AssertionError.class);
     assertThatThrownBy(
-            () ->
-                assertThat(LOG_DATA)
-                    .hasAttributesSatisfying(equalTo(AttributeKey.stringKey("bear"), "moo")))
+            () -> assertThat(LOG_DATA).hasAttributesSatisfying(equalTo(stringKey("bear"), "moo")))
         .isInstanceOf(AssertionError.class);
     assertThatThrownBy(
             () ->
                 assertThat(LOG_DATA)
                     .hasAttributesSatisfyingExactly(
-                        equalTo(AttributeKey.stringKey("bear"), "mya"),
+                        equalTo(stringKey("bear"), "mya"),
                         equalTo(AttributeKey.booleanKey("warm"), true),
-                        equalTo(AttributeKey.longKey("temperature"), 30L),
+                        equalTo(longKey("temperature"), 30L),
                         equalTo(AttributeKey.doubleKey("length"), 1.2)))
         .isInstanceOf(AssertionError.class);
     assertThatThrownBy(() -> assertThat(LOG_DATA).hasTotalAttributeCount(11))
         .isInstanceOf(AssertionError.class);
   }
+
+  @Test
+  void eventBodyAssertions() {
+    InMemoryLogRecordExporter exporter = InMemoryLogRecordExporter.create();
+    SdkLoggerProvider loggerProvider =
+        SdkLoggerProvider.builder()
+            .addLogRecordProcessor(SimpleLogRecordProcessor.create(exporter))
+            .build();
+    EventLogger eventLogger = SdkEventLoggerProvider.create(loggerProvider).get("test.test");
+    eventLogger
+        .builder("foo")
+        .put("foostr", "bar")
+        .put("foobool", true)
+        .put("foolong", 12L)
+        .put("foodbl", 12.0)
+        .put("foostra", "bar", "baz", "buzz")
+        .put("foolonga", 9, 0, 2, 1, 0)
+        .put("foodbla", 9.1, 0.2, 2.3, 1.4, 0.5)
+        .put("fooboola", true, true, true, false)
+        .put("fooany", AnyValue.of("grim"))
+        .put(stringKey("ak_str"), "bar")
+        .put(booleanKey("ak_bool"), true)
+        .put(longKey("ak_long"), 12L)
+        .put(doubleKey("ak_dbl"), 12.0)
+        .put(stringArrayKey("ak_stra"), Arrays.asList("bar", "baz", "buzz"))
+        .put(longArrayKey("ak_longa"), Arrays.asList(9L, 0L, 2L, 1L, 0L))
+        .put(doubleArrayKey("ak_dbla"), Arrays.asList(9.1, 0.2, 2.3, 1.4, 0.5))
+        .put(booleanArrayKey("ak_boola"), Arrays.asList(true, true, true, false))
+        .emit();
+    List<LogRecordData> logs = exporter.getFinishedLogRecordItems();
+    assertThat(logs).hasSize(1);
+    assertThat(logs.get(0))
+        .hasBodyField("foostr", "bar")
```

Comment:
What do you think about naming these `containsEntry` to mirror the naming of [AttributeAssert](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk/testing/src/main/java/io/opentelemetry/sdk/testing/assertj/AttributesAssert.java)?

On one hand, a different name helps differentiate body from attributes. But `containsEntry` seems to mirror the vocabulary for java map, and both attributes and body (commonly) are map-like structures, so using the same language in both cases feels appropriate.

Reply:
My opinion is that `hasBodyField` is a little more precise and a casual reader could (wrongly) assume that `containsEntry` checks attributes.

Reply:
👍